dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -60,7 +60,7 @@
 			};
 		};
 
-		flexspi0: flexspi0@402a8000 {
+		flexspi0: spi0@402a8000 {
```

Comment:
why is this instance `spi0` and the other one just `spi`?

Reply:
Should have been `spi`.  Fixed now.